Tree.js

```diff
@@ -63,6 +63,7 @@ define([
                     layerData = _.assign({}, serviceLayer || {}, layerDetails || {}),
                     layerId = this.createLayerId(parent, layerData),
                     node = _.assign(layerData, {
+                        includeAllLayers: !!serviceLayer.subLayerIds,
```

Comment:
From reading the code, it's not clear why this is a computed property and not using the `includeAllLayers` flag from the config.  If it's actually different, maybe using a different key name would be less ambiguous.